### ISSUE TITLE
slides building feedback for Dorai

### ISSUE THREAD

**Issue by retabak** [bootstrapworld/curriculum#1670]
to be provided as I build udpated slide decks for early math

**Comment by ds26gte** [comment]
Hmm. How does it make semantic sense to have a lower-level content divider inside a higher-level one? (It's like having a section inside a subsection, no?)

**Comment by schanzer** [comment]
@retabak Dorai and I spoke and worked out a MUCH nicer solution.

Back-to-back `lesson-instruction`, `QandA`, and `lesson-roleplay` blocks will now show up as if they're one big one in lesson plans. So you can write...

```
@lesson-instruction{
Do this
}
@slidebreak
@lesson-instruction{
Do this
}
```

It will show up as if it's a single block in the lesson plan, but as separate slides!

If this answers your questions, feel free to close the issue!

**Comment by retabak** [comment]
if we want additional headings in the slides (that aren't in the lesson), I assume we're just out of luck?

**Comment by schanzer** [comment]
@retabak Correct. You can, of course, always tweak the generated .md file or the slides themselves by hand, the way we always have. The idea isn't that this reduces the amount of slide work _zero_ -- just a lot closer to it.
